pkg/buffer/buffer.go

```diff
@@ -145,13 +158,13 @@ func (b *Buffer) Bind(params webrtc.RTPParameters, o Options) {
 		for _, fb := range codec.RTCPFeedback {
 			switch fb.Type {
 			case webrtc.TypeRTCPFBGoogREMB:
-				log.Debugf("Setting feedback %s", webrtc.TypeRTCPFBGoogREMB)
+				debugLogger.Info("Setting feedback", "type", "webrtc.TypeRTCPFBGoogREMB")
```

Comment:
```suggestion
				Logger.V(1).Info("Setting feedback", "type", "webrtc.TypeRTCPFBGoogREMB")
```

Here and elsewhere, just consume the package global logger.

Reply:
Will do the same for sfu

Reply:
Added